---
layout: post
issue: "2022"
category: feature
order: 9
title: If genes could talk
subhead: "Genomic sleuthing paints pictures of the past—and present"
author-last: Williams
banner:
  file: 2022/if-genes-could-talk-banner.jpg
  position: left
further-inquiry:
    - title: "Gorzynski J, Shafin K, Jensen T, et al. Ultrarapid Nanopore Genome Sequencing in a Critical Care Setting. <em>New England Journal of Medicine.</em> 2022 Feb 17;386(7):700-702."
      url: https://www.nejm.org/doi/10.1056/NEJMc2112090
    - title: "van der Valk T, Pečnerová P, Díez-del-Molino D, et al. Million-year-old DNA sheds light on the genomic history of mammoths.<em>Nature</em> 2021;591:265-69"
      url: https://www.nature.com/articles/s41586-021-03224-9
    - title: "Schaefer N, Shapiro B, Green R. An ancestral recombination graph of human, Neanderthal, and Denisovan genomes. <em>Science Advances</em> 2021; 7:29."
      url: https://www.science.org/doi/10.1126/sciadv.abc0776
    - title: "Vershinina A, Heintzman P, Froese D, et al. Ancient horse genomes reveal the timing and extent of dispersals across the Bering Land Bridge. <em>Molecular Ecology</em>"
      url: https://onlinelibrary.wiley.com/doi/10.1111/mec.15977
    - title: "2021;30:6144-61."
      url: https://onlinelibrary.wiley.com/doi/10.1111/mec.15977
    - title: "Saremi N, Supple M, Byrne A, et al. Puma genomes from North and South America provide insights into the genomic consequences of inbreeding. <em>Nature Communications</em> 2019; 10:4769."
      url: https://www.nature.com/articles/s41467-019-12741-1
       
in-the-news:
    - title: "First complete, gapless sequence of a human genome reveals hidden regions (UCSC News, March 2022)."
      url: https://news.ucsc.edu/2022/03/t2t-genome.html
    - title: "A more-inclusive genome project aims to capture all of human diversity (<em>Nature</em>, March 2022)."
      url: https://www.nature.com/articles/d41586-022-00726-y
    - title: "Astra Forensics Makes the Untraceable Traceable (<em>Santa Cruz Sentinel</em>, November 2021)."
      url: https://www.santacruzsentinel.com/2021/11/18/made-in-santa-cruz-astrea-forensics-makes-the-untraceable-traceable/
    - title: "If we want to save the natural world, we're going to have to change it (<em>Washington Post</em>, November 2021)."
      url: https://www.washingtonpost.com/outlook/if-we-want-to-save-the-natural-world-were-going-to-have-to-change-it/2021/11/05/1c882f0e-38ed-11ec-9bc4-86107e7b0ab1_story.html
    - title: "Million-year-old mammoth genomes shatter record for oldest ancient DNA (<em>Nature</em>, February 2021)."
      url: https://www.nature.com/articles/d41586-021-00436-x
    - title: "The Ancient Neanderthal Traces Hidden in Your Genome (<em>Science Friday</em>, October 2021)."
      url: https://www.sciencefriday.com/segments/neanderthal-human-genome/
    - title: "Saving Pumas with Genomics (<em>Cosmos</em>, October 2019)."
      url: https://cosmosmagazine.com/earth/sustainability/saving-pumas-with-genomics/
    - title: "Why This Scientist Keeps Receiving Packages of Serial Killers' Hair (<em>New York Times</em>, October 2019)."
      url: https://www.nytimes.com/2019/09/16/science/hair-dna-murder.html
---
{% include_relative sidebars/if-genes-could-talk-hero-caption.html classes="right" bgcolor="#f1f1f1" %}

More than a million years ago, snow and ice buried a [mammoth](https://www.britannica.com/animal/mammoth-extinct-mammal)'s decaying body in Siberia. Over time, most of the "Krestovka mammoth" vanished; first its soft tissues and later even its bones became food for bacteria and bugs. Some teeth remained, however, and a Russian paleontologist unearthed a few in 1970. For decades, they sat in museum collections as curiosities, too old to carbon date. Scientists presumed any lingering organic material long gone.

However, in [research published February 2021](https://www.nature.com/articles/d41586-021-00436-x) in the scientific journal *Nature*, an international team of scientists including [**Beth Shapiro**](https://www.eeb.ucsc.edu/about/dept-directory.php?uid=bashapir), professor of ecology and evolutionary biology, turned to modern [genomics](https://www.genome.gov/about-genomics/fact-sheets/A-Brief-Guide-to-Genomics) to reveal more about this ancient mammoth and its relatives. Drilling a miniscule amount of material---about a pinch of salt's worth---out of one molar, the team was able to isolate tattered bits of [DNA](https://medlineplus.gov/genetics/understanding/basics/dna/) from the sample. Their [sequencing](https://www.nature.com/articles/s41586-021-03224-9) of this and other mammoth DNA showed that the Krestovka specimen represented a previously unknown genetic lineage of mammoths, upending previous hypotheses about mammoth evolution. Based on its sequenced genes and how much they varied from those of more modern mammoths, the team estimated that the animal lived about 1.2 million years ago. Before this effort, no one had ever sequenced genes more than a million years old.

Until recently, such a genomic dive into the deep past was considered impossible. But gene sequencing continues to get better, faster, and cheaper, allowing scientists, like Shapiro and others at UCSC, to develop new ways to piece together and analyze genes from even the most fragmented DNA specimens. The advances are letting researchers solve long-standing mysteries about evolution, migration, and diversity---in both animals and humans. They're also a boon for understanding and diagnosing genetic diseases and helping criminal investigators solve cold cases. "Even a single genome can tell us all kinds of things about the broader population it belongs to," said Shapiro. "It really can paint a picture of the past."

## Dog-chewed ##

<figure class="left" style="width:300px;">
  {% asset 2022/bison-hide-moccasin.jpg @magick:resize=300 alt:'Photo: Bison hide moccasin' %}<figcaption markdown="span">Researchers in the UCSC Paleogenomics Lab analyzed DNA from 38 bison hide moccasins, like this one, [worn more than 800 years ago](https://nhmu.utah.edu/blog/2016/08/23/promontory-culture) by cave dwellers in what is now Utah. They discovered that nearly all of the moccasins, which were well preserved in dry caves for centuries, [came from the hides of female bison](https://www.sciencedirect.com/science/article/pii/S0305440321002077). The researchers hypothesized that the Indigenous people of the American plains may have purposefully targeted female bison for moccasin manufacture. Credit: [Daderot](https://commons.wikimedia.org/wiki/File:Promontory_moccasin,_Promontory_Cave_I,_Utah,_1225-1275_AD,_bison_hide_and_hair_pad_-_Natural_History_Museum_of_Utah_-_DSC07382.JPG) (public domain).</figcaption>
</figure>
When people swab the inside of their cheeks with test kits like those from [23andMe](https://www.23andme.com/) and [ancestry.com](https://www.ancestry.com/), they collect---among other things---strands of genetic material, each containing hundreds of millions of relatively intact [nucleotide base pairs](https://www.genome.gov/genetics-glossary/Base-Pair), the building blocks of DNA. Sequencing such DNA samples is straightforward. But immediately after an organism dies, the natural world around them starts breaking down their components, including their DNA. Further muddling the genomic picture, microbes and other decaying plants and animals seep into the heap, adding their DNA remnants.

If sequencing DNA from a living person is like putting together a thousand-piece puzzle per the picture on the box cover, then sequencing ancient DNA is like solving such a puzzle after your dog chews up each puzzle piece, someone mixes those pieces with dog-chewed pieces from several other puzzles, and you lose the box. "To sequence ancient genomes, we need to take dirty, crumbled up DNA and transform it into molecules that can be sequenced," said Shapiro.

Shapiro and her husband [**Richard Edward Green**](https://engineering.ucsc.edu/people/ed), associate professor of biomedical engineering, now routinely perform this difficult task in the [UCSC Paleogenomics Laboratory](https://pgl.soe.ucsc.edu/) they run together. The work involves tedious washing, isolating, and handling to get rid of contaminants, gene amplification to make millions of copies of tiny bits of DNA, and powerful computer programs that assemble the short sequences into more complete gene readouts. "We need specialized facilities where we don't touch our samples," said Shapiro. "We don't even breathe on them---if our DNA gets into this ancient DNA confetti, it makes our experiments much harder."

## Assessing admixing ##

<figure class="left" style="width:300px;">
  {% asset 2022/mammoth-tooth.jpg @magick:resize=300 alt:'Photo: Mammoth tooth' %}<figcaption markdown="span">DNA from mammoth teeth, like the one shown here in the Palatinate Museum of Natural History in Germany, let scientists at the UCSC Paleogenomics Lab and their collaborators discover how two different lineages of mammoths interbred to create an entirely new species of mammoth more than 420,000 years ago. Credit: [BlueBreezeWiki](https://commons.wikimedia.org/wiki/File:150322-Mammutzahn.jpg) (CC BY-SA 3.0).</figcaption>
</figure>
Work at the Paleogenomics Laboratory in the last few years has sequenced not only the genomes of mammoths but also those of ancient [bears](https://doi.org/10.1016/j.cub.2021.04.027), [dogs](https://www.nature.com/articles/s41586-020-03082-x), [bison](https://doi.org/10.1093/jhered/esab003), [horses](https://www.nature.com/articles/s41586-021-04018-9), and [Neanderthals](https://advances.sciencemag.org/content/7/29/eabc0776). A key question tying many of the projects together is what genetically defines a [species](https://www.newscientist.com/definition/species/). "As humans, we have this proclivity to put things into boxes---a polar bear is a polar bear, and a brown bear is a brown bear," said Shapiro. But this view is too simple. "Speciation typically doesn't happen overnight."

In the genomes of nearly all the ancient animals Shapiro and others have studied, the results indicate that those from different lineages often interbred even after beginning to separate into distinct species, a phenomenon called [admixing](https://en.wikipedia.org/wiki/Genetic_admixture). The result? Brown bears [have snippets of polar bear DNA](https://www.pnas.org/doi/10.1073/pnas.1210506109), bison [have cattle in their genomes](https://www.nytimes.com/2002/04/23/science/genetically-bison-don-t-measure-up-to-frontier-ancestors.html), and modern humans share a surprisingly large amount of DNA with Neanderthals.
<figure class="right" style="width:300px;">
  {% asset 2022/puma.jpg @magick:resize=300 alt:'Photo: puma on a tree limb' %}<figcaption markdown="span">By comparing the genomes of ten [pumas](https://animals.sandiegozoo.org/animals/mountain-lion-puma-cougar) from sites across North and South America, Beth Shapiro, Ed Green and their colleagues at the UCSC Paleogenomics Laboratory assembled a puma family tree. North American pumas, they calculated, diverged from their South American relatives [more than 100 thousand years ago](https://www.nature.com/articles/s41467-019-12741-1). The results also showed that Florida panthers, like the one seen here, remain inbred even after cross-breeding efforts in the 1990s. Credit: [U.S. Army Corp of Engineers](https://commons.wikimedia.org/wiki/File:Puma_concolor_coryi_Florida_panther.jpg) (CC BY 2.0).</figcaption>
</figure>
Cataloguing these genetic admixtures can help scientists understand how ancient populations arose and declined. But it also can prove valuable in conservation efforts to help save struggling species from extinction. In some cases, preventing admixing might be critical to ensuring the survival of a rare species---if two species interbreed too frequently, the unique traits of one may completely fade away. More often, though, endangered populations suffer from interbreeding between close relatives, which can harm a population by increasing the prevalence---and impact---of detrimental genes.

To make at-risk species healthier and more likely to survive long term, conservationists sometimes encourage the breeding of isolated populations with closely related animals from the outside. Scientists [successfully boosted](https://www.tampabay.com/environment/breeding-florida-panthers-with-cougars-saved-the-state-animal-from-extinction-new-study-says-20190719/) Florida's dwindling panther population in the 1990s, for instance, by crossbreeding the Florida animals with Texas cougars. Having genetic information on each species helps conservationists choose which populations---or even individual animals---to breed with each other, which can be critical to the success of such efforts. "The past is a completed evolutionary experiment," said Shapiro. "But when we try to devise strategies to protect species, we\'re making a lot of guesses. By looking into the past, we can better understand what has already worked."

## Seeing differences ##

The genomic science advances enabling scientists to study ancient DNA and help endangered animals also have vast promise to improve human health, said [**Benedict Paten**](https://engineering.ucsc.edu/people/benedict), associate professor of biomolecular engineering and associate director of the [UCSC Genomics Institute](https://genomics.ucsc.edu/).
{% include_relative sidebars/coming-to-terms.html classes="left" bgcolor="#f1f1f1" %}Some of that promise is clearcut. For example, as [reported](https://www.nejm.org/doi/full/10.1056/NEJMc2112090) in February 2022 in the *New England Journal of Medicine*, Paten and collaborators showed that rapid sequencing of the entire genomes of critically ill patients uncovered genetic variations that impacted the treatment of five of the dozen subjects enrolled in the study. But other implications are less straight-forward. Understanding human genetic evolution, Paten said, can shed light on how today's diversity arose and how it impacts human health, which could help address the increasingly acknowledged---yet quite elusive---problem of systemic racial bias in medicine.

Paten chairs the Genome Alignment and Annotation Committee of the [Vertebrate Genomes Project](https://vertebrategenomesproject.org/), a group of more than 150 scientists, each with their own funding, [aiming to collaboratively sequence](https://www.hhmi.org/news/project-to-read-genomes-of-all-70000-vertebrate-species-reports-first-discoveries?utm_source=twitter&utm_medium=social&utm_content=100002026866726&utm_campaign=hhmi-campaign) at least one genome from 66,000 vertebrate species, including mammals, birds, reptiles, amphibians, and fish. Like Shapiro and Green, Paten is not only interested in the stand-alone genomes of single species, but how these genomes compare and have evolved and diverged over time. "On their own, genomes can be incredibly interesting but they're also snapshots in time in a long evolutionary history," Paten said. Comparing the genomes of different species can reveal new genes and proteins that could lead to the discovery and development of a broad range of innovative items, from medicines to adhesives or even electronics, said Paten. "Every species contains unique genes that can be powerful in terms of understanding biology and giving us new tools."

In addition, understanding how and why one gene has evolved to vary between species while another has remained stubbornly stagnant can tell Paten and other researchers much about the importance---and flexibility---of that gene. Paten gauges this flexibility by comparing variation between modern species, while Shapiro and researchers in the Paleogenomics Lab often try to assess the same thing by analyzing the genomes of ancient species. In July 2021, for instance, the journal *Scientific Advances* published Shapiro and Green's research comparing modern human and Neanderthal genomes to see which parts of the modern human genome were never---in any individuals---derived from Neanderthal ancestry. [Less than 7% of the modern human genome](https://www.science.org/doi/10.1126/sciadv.abc0776?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed), they discovered, is unique and never of Neanderthal origin. "Some traits move between species and become advantageous, but others don't, because they can't. They're not compatible," said Shapiro, adding that knowledge of a gene's natural variation could eventually prove helpful to bioengineers attempting to repair or improve its function.
<figure class="right" style="width:600px;">
  {% asset 2022/horse-bones.jpg @magick:resize=600 alt:'Photo: Horse bones from the Ice Age' %}<figcaption markdown="span">By sequencing ancient DNA from these bones of horses that lived in North America during the last Ice Age, scientists at the UCSC Paleogenomics Lab shed new light on [how different horse populations diverged](https://pgl.soe.ucsc.edu/horses.html)---and then later crossed paths again. Eurasian and North American horses, they discovered, crossed the Bering Land Bridge repeatedly over the course of hundreds of thousands of years. Understanding how, when, and where these populations evolved has implications for managing today's wild horse populations. Credit: Alisa Vershinina/Paleogenomics Lab, with permission.</figcaption>
</figure>

## Pushing envelopes ##

{% include_relative sidebars/csi-santa-cruz.html classes="left" bgcolor="#f1f1f1" %}
If genomes could talk, each human's would sound different. Diversity in ancestry, race, disease risk, metabolism, and a plethora of other features, from personality to appearance, are all captured in our DNA. But for the past nearly twenty years since the milestone completion of the [Human Genome Project](https://www.genome.gov/human-genome-project) in 2003, researchers have used a single reference genome---based almost entirely on [the genome of a single individual](https://www.statnews.com/2019/03/11/human-reference-genome-shortcomings/) from Buffalo, New York---as the template for all other human genomes. "That's a problem," said [**Karen Miga**](https://engineering.ucsc.edu/people/khmiga), Paten's colleague, and an assistant professor of biomedical engineering. "The reference genome gives us only one snapshot of a human genome when we all know that our genomes differ, sometimes in very large ways."

When researchers analyze a person's genome, they use that reference genome as a point of comparison. Places where an individual's genes vary greatly from the reference may indicate causes of disease, for instance. But for some populations, the current reference genome may not be a fair place to start---it may not capture the norm at all.

Like Shapiro, Green, and Paten, Miga is pushing the envelope on how to obtain the most detailed information from genomes and capture the differences between them. Miga co-chaired the [Telomere-to-Telomere consortium](https://sites.google.com/ucsc.edu/t2tworkinggroup), an effort---[just recently completed](https://news.ucsc.edu/2022/03/t2t-genome.html) and reported in six papers published in the March 31 issue of *Science*---to fill in the gaps that remained in the original reference genome. She also co-chairs working groups within the broader [Human Pangenome Project](https://humanpangenome.org/about-us/consortium-organization/), which aims to create a more global reference genome that better represents human genomic diversity. In all this work, Miga [uses multiple new technologies, including nanopore sequencing](https://www.youtube.com/watch?v=CcNCMICY-Ls&t=202s), that let researchers piece together much larger chunks of the genome at a time---akin to completing a puzzle with a hundred pieces instead of one with a thousand pieces. This makes it easier to figure out the many repetitive stretches of DNA within people's genomes---a challenge that's a bit like assembling a large blue sky of a puzzle.

The rapidly improving technology, Miga said, doesn't just benefit the Pangenome Project, but gene sequencing for all other purposes. This broad applicability rings true for all the researchers. The same breakthroughs that allow them to determine how horses or wolves evolved can also help them better diagnose patients with rare diseases, identify criminals, and engineer improved food crops. "A lot of basic research has a very long tail," said Paten. "Genomics often asks some fairly technical questions about how we can assemble the whole genome and all its variations, but the progress we make on those questions serves us well in many, many other areas."
